<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PurchasForm</class>
 <widget class="QWidget" name="PurchasForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>826</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLineEdit#le_merchname{
color: rgb(255, 0, 0);
background-color:transparent;
border-width:0;border-style:outset
}
QLineEdit{
color: rgb(0, 0, 0);
font: Bold;
background-color:transparent;
border-width:0;border-style:outset
}
QTextEdit{
color: rgb(0, 0, 0);
background-color:transparent;
border-width:0;border-style:outset
}
QPushButton{	
	
	background-color: rgb(255, 116, 116);
    border-style: outset;
	border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    font: bold 25px;
	
	selection-color: rgb(0, 170, 255);

    padding: 6px;
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_12">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_11">
     <item row="0" column="0">
      <widget class="QGroupBox" name="groupBox_3">
       <property name="title">
        <string>商品详情</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_10">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout_9">
          <item row="0" column="0" colspan="2">
           <widget class="QLineEdit" name="le_merchname">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>35</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Adobe Devanagari</family>
              <pointsize>14</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="ib_hostPhoto">
            <property name="minimumSize">
             <size>
              <width>200</width>
              <height>200</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>200</width>
              <height>200</height>
             </size>
            </property>
            <property name="text">
             <string>=========加载失败=========</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <layout class="QGridLayout" name="gridLayout_8">
            <item row="0" column="0">
             <layout class="QGridLayout" name="gridLayout_7">
              <item row="0" column="2" rowspan="3">
               <spacer name="horizontalSpacer">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeType">
                 <enum>QSizePolicy::Maximum</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>10</width>
                  <height>80</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item row="0" column="3" colspan="3">
               <widget class="QLineEdit" name="le_storename">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>150</width>
                  <height>0</height>
                 </size>
                </property>
               </widget>
              </item>
              <item row="1" column="1" rowspan="2">
               <widget class="QLCDNumber" name="lcdNumber_price">
                <property name="minimumSize">
                 <size>
                  <width>150</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>150</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <weight>75</weight>
                  <bold>true</bold>
                  <underline>false</underline>
                 </font>
                </property>
                <property name="mode">
                 <enum>QLCDNumber::Dec</enum>
                </property>
                <property name="segmentStyle">
                 <enum>QLCDNumber::Flat</enum>
                </property>
               </widget>
              </item>
              <item row="0" column="0">
               <widget class="QLabel" name="label_2">
                <property name="minimumSize">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
                <property name="text">
                 <string>商店：</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QLineEdit" name="le_storeID">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>150</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>150</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
              <item row="1" column="0" rowspan="2">
               <widget class="QLabel" name="label_3">
                <property name="minimumSize">
                 <size>
                  <width>40</width>
                  <height>50</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>40</width>
                  <height>50</height>
                 </size>
                </property>
                <property name="text">
                 <string>价格：</string>
                </property>
               </widget>
              </item>
              <item row="2" column="3">
               <widget class="QLabel" name="label_7">
                <property name="maximumSize">
                 <size>
                  <width>40</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="text">
                 <string>尺码：</string>
                </property>
               </widget>
              </item>
              <item row="1" column="3">
               <widget class="QLabel" name="label_8">
                <property name="maximumSize">
                 <size>
                  <width>40</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="text">
                 <string>数量：</string>
                </property>
               </widget>
              </item>
              <item row="1" column="4">
               <widget class="QSpinBox" name="sB_num">
                <property name="maximumSize">
                 <size>
                  <width>50</width>
                  <height>16777215</height>
                 </size>
                </property>
               </widget>
              </item>
              <item row="2" column="4">
               <widget class="QComboBox" name="cb_size">
                <property name="maximumSize">
                 <size>
                  <width>50</width>
                  <height>16777215</height>
                 </size>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item row="1" column="0">
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Maximum</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>388</width>
                <height>10</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="2" column="0">
             <layout class="QGridLayout" name="gridLayout_6">
              <item row="1" column="0">
               <widget class="QLineEdit" name="le_stock">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>60</width>
                  <height>16777215</height>
                 </size>
                </property>
               </widget>
              </item>
              <item row="0" column="2" rowspan="2">
               <widget class="QPushButton" name="pb_purchase">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>60</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>180</width>
                  <height>60</height>
                 </size>
                </property>
                <property name="text">
                 <string>立即购买</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1" rowspan="2">
               <widget class="QPushButton" name="pb_addTerolly">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>60</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>180</width>
                  <height>60</height>
                 </size>
                </property>
                <property name="text">
                 <string>加入购物车</string>
                </property>
               </widget>
              </item>
              <item row="0" column="0">
               <widget class="QLabel" name="label_6">
                <property name="maximumSize">
                 <size>
                  <width>16677</width>
                  <height>15</height>
                 </size>
                </property>
                <property name="text">
                 <string> 库存：</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QGroupBox" name="groupBox_4">
       <property name="title">
        <string>图片信息</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="1" column="0">
         <widget class="QListWidget" name="listWidget_2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Ignored">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>395</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QTextEdit" name="te_descri">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>50</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>50</height>
           </size>
          </property>
          <property name="html">
           <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
